src/data.cpp

```diff
@@ -166,7 +173,7 @@ double pyne::natural_abund(int nuc) {
   // state abundance...not strictly true, but good guess.
   if (0 < nucid%10000) {
     na = natural_abund((nucid/10000)*10000);
-    atomic_mass_map[nuc] = na;
+    natural_abund_map[nuc] = na;
```

Comment:
I don't think it needs to be since we already have the value (ie `na`). The trouble with indexing maps really comes when you access without setting.